src/stormpath.social-login.linkedin.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+angular.module('stormpath.linkedinLogin', [])
+.provider('$linkedinLogin', function() {
+  function loadLinkedinSDK(clientId, onload) {
+    var id = 'linkedin-jssdk';
+    var firstJs = document.getElementsByTagName('script')[0];
+    var js;
```

Comment:
`js` is very unclear. I'd name it something more like clear like `scriptElement`.